quickstart/install.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+set -e
+
+# Installing Laravel message
+printf "\n+ Installing Laravel (This may take a few minutes)...\n"
+
+# Download the Laravel installer
+composer require "laravel/installer"
+
+# Create a new Laravel app
+laravel new
+
+# Update the database connection variables in .env
+sed -i -e "
+s/DB_HOST.*$/DB_HOST=\$DATA_MYSQL_HOST/g;
+s/DB_DATABASE.*$/DB_DATABASE=gonano/g;
+s/DB_USERNAME.*$/DB_USERNAME=\$DATA_MYSQL_USER/g;
+s/DB_PASSWORD.*$/DB_PASSWORD=\$DATA_MYSQL_PASS/g;
+s/REDIS_HOST.*$/REDIS_HOST=\$DATA_REDIS_HOST/g
+" .env
+
+# Generate .env.prod
+cp .env .env.prod
+
+# Documentation to be prepended to .env.prod
+documentation="# PRODUCTION ENVIRONMENT SETTINGS
+# When deploying to a live app with Nanobox, this file will
+# replace your local .env file. Sensitive values should not
+# be included here. These values should be added through your
+# application dashboard or with the \`nanobox evar add\` command.\n
+# https://docs.nanobox.io/app-config/environment-variables/#custom-environment-variables\n
+# * Should be added in your dashboard or through the nanobox cli\n\n"
+
+# Prepend .env.prod with documentation
+echo -e "$documentation$(cat .env.prod)" > .env.prod
```

Comment:
I'd drop the `cp .env .env.prod` line, and use `cat .env` instead, here. Avoids race conditions from opening the redirect before the subshell, and has the same effect as the copy-then-modify is intended to.

Reply:
Ah, great thought.